test/UnitTest/Microsoft.AspNetCore.OData.Test/Abstraction/RoutingConfigurationFactory.cs

```diff
@@ -157,6 +157,67 @@ public static IRouteBuilder CreateWithRootContainerAndTypes(string routeName = n
 
             return builder;
         }
+
+        /// <summary>
+        /// Initializes a new instance of the routing configuration class.
+        /// </summary>
+        /// <returns>A new instance of the routing configuration class.</returns>
+        public static IRouteBuilder CreateWithCaseInsensitiveModelBinding()
+        {
+            IServiceCollection serviceCollection = new ServiceCollection();
+            serviceCollection.AddMvc();
+            serviceCollection.AddSingleton<ObjectPoolProvider, DefaultObjectPoolProvider>();
+            serviceCollection.AddSingleton<ILoggerFactory>(NullLoggerFactory.Instance);
+            serviceCollection.AddOData();
+            serviceCollection.AddSingleton(new ODataOptions(){ EnableCaseInsensitiveModelBinding = true});
```

Comment:
why don't use call the setting extension method after "AddOData"

Reply:
#Resolved